tools/blocksconvert/scanner/scanner.go

```diff
@@ -204,27 +204,30 @@ func (s *Scanner) running(ctx context.Context) error {
 
 	level.Info(s.logger).Log("msg", "total found tables", "count", len(allTables))
 
+	if s.cfg.TableNames != "" {
+		// Find tables from parameter.
+		tableNames := map[string]bool{}
+		for _, t := range strings.Split(s.cfg.TableNames, ",") {
+			tableNames[strings.TrimSpace(t)] = true
+		}
+
+		for ix := 0; ix < len(allTables); {
```

Comment:
`ix` is never increased

Reply:
It is, on line 223. (There was bug before, perhaps you have seen buggy version of the code)

Reply:
Yes, I reviewed the previous version.